spec/models/user_session_spec.rb

```diff
@@ -125,6 +125,36 @@
     end
   end
 
+  describe '#planned_processing' do
+    it 'returns nil if the key is not on the session' do
+      expect(user_session.planned_processing).to be nil
+    end
+
+    context 'when the key is present on the session' do
+      let(:session) do
+        {
+          Wizard::Steps::PlannedProcessing.id => 'yes',
+        }
+      end
+
+      let(:expected_response) { 'yes' }
+
+      it 'returns the date from the session' do
+        expect(user_session.planned_processing).to eq(expected_response)
+      end
+    end
+  end
+
+  describe '#planned_processing=' do
+    let(:expected_response) { '1' }
+
+    it 'sets the key on the session' do
+      user_session.planned_processing = '1'
+
```

Comment:
We decided to use yes/no everywhere instead of 1/0s.

Reply:
Also, I don't think this is quite right. I think you should be storying the type of processing: `without_any_processing`,  `annual_turnover, ..etc. The ones that you defined at the model level under OPTIONS?
so the session would I think look like e.g: 
```ruby
{..., planned_processing: 'without_any_processing' ,...}
```